src/controllers/user.controller.js

```diff
@@ -0,0 +1,15 @@
+const {getAll} = require('../db/queries/user/read');
+const authMiddleware = require('../middlewares/authentication.middleware');
+
+module.exports = {
+    getAllUsers(req, res) {
+        const auth = new authMiddleware(req, res);
+
+        if (auth.checkToken()) {
+            res.json({
```

Comment:
As we talked about before, I think It's better to use services because from the query we receive "Promise" so we need `.then()` and `.catch()` here (you can see the examples in tarin or auth controller).
Also if `getAll()` has error we always see { }, but we need to response a related `message`.
![Screenshot (653)](https://user-images.githubusercontent.com/49534520/194931283-583d40f6-c4be-49bc-a625-084aeeaed7a1.png)